constellations/gemini-user/src/internship/internship.sass

```diff
@@ -20,26 +25,47 @@
         & img
             width: 120px
             margin-right: 60px
-        
+    
         & p
             margin: 0
-     
 
-    #past-intern
-        border-collapse: collapse
-        border-spacing: 2px
-        width: 80%
-        margin: 30px auto
+    .intern-main
+        display: flex
+        flex-direction: row
+        & p
+            margin: 20px 20px 20px 0
```

Comment:
Just give this a these a bottom and right margin `margin: 0 20px 20px 0`. I think the interns section takes up too much room.